autotest/cpp/makefile.vc

```diff
@@ -32,7 +32,8 @@ GDAL_SUPPORT = \
     -DSHAPE_ENABLED \
     -DFRMT_aaigrid \
     -DFRMT_dted \
-    -DFRMT_gtiff
+    -DFRMT_gtiff \
+	-DFRMT_jpegxr
```

Comment:
-DFRMT_jpegxr should only be defined if the library has JPEGXR support. Otherwise tests will fail